ui/hooks/useIncrementedGasFees.js

```diff
@@ -68,19 +66,19 @@ export function useIncrementedGasFees(transactionGroup) {
     // do not have txParams. This is why we use optional chaining on the
     // txParams object in this hook.
     const temporaryGasSettings = {
-      gasLimit: primaryTransaction.txParams?.gas,
-      gas: primaryTransaction.txParams?.gas,
+      gasLimit: transaction?.txParams?.gas,
```

Comment:
Is it possible for the `transaction` to be falsy at this point? I think all of these `?`'s can be dropped. This hook doesn't make a lot of sense if there's no transaction.

Reply:
Indeed! Removed in latest commit: [902279a](https://github.com/MetaMask/metamask-extension/pull/12069/commits/902279ae2d23f645eb764704bec5b9f6512a6231)